Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -107,6 +107,13 @@ void AsyncScrollingCoordinator::handleWheelEventPhase(ScrollingNodeID nodeID, Pl
 }
 #endif
 
+RefPtr<ScrollingStateNode> AsyncScrollingCoordinator::stateNodeForNodeID(ScrollingNodeID nodeID) const
+{
+    if (RefPtr scrollingNode = m_scrollingStateTree->stateNodeForID(nodeID))
```

Comment:
We could just return this for now instead of null checking then returning.

Reply:
Fixed.